### PR TITLE
Reword README

### PR DESCRIPTION
**Pull request by Jayshua**
As a native English speaker, I thought I'd reword the Readme a bit. Let me know if there are any changes you don't like.

### CODE REVIEW

**Comment by arowM** [comment]
Thank you very much!
It’s super helpful, I can learn a lot from you!

**Comment by Jayshua** [comment]
Just noticed that my last set of changes didn't make it into this merge request for some reason... Don't learn too much from this yet - it has a few grammar mistakes :)
